src/components/ScreenWrapper.js

```diff
@@ -9,8 +10,11 @@ const propTypes = {
     // Array of additional styles to add
     style: PropTypes.arrayOf(PropTypes.object),
 
-    // Returns a function as a child to pass insets to
-    children: PropTypes.func.isRequired,
+    // Returns a function as a child to pass insets to or a node to render without insets
```

Comment:
I love how this solution turned out. It's much better than the dual components that I suggested.